File: Parcial1/areaTriangulo.py
# Programa para calcular el área de un triángulo

try:
    # Solicitar la base y la altura
    base = float(input("Ingresa la base del triángulo: "))
    altura = float(input("Ingresa la altura del triángulo: "))

    # Validar la entrada
    if base <= 0 or altura <= 0:
        print("Error: La base y la altura deben ser números positivos.")
    else:
        # Calcular el área
        area = (base * altura) / 2

        # Mostrar el resultado
        print(f"El área del triángulo con base {base:.2f} y altura {altura:.2f} es: {area:.2f}")

except ValueError:
    print("Error: Debes ingresar valores numéricos.")

File: Parcial1/areaTriangulo.m
% Programa para calcular el área de un triángulo

% Solicitar la base y la altura
base = input('Ingresa la base del triángulo: ');
altura = input('Ingresa la altura del triángulo: ');

% Validar la entrada
if ~isnumeric(base) || base <= 0
    fprintf('Error: La base debe ser un número positivo.\n');
elseif ~isnumeric(altura) || altura <= 0
    fprintf('Error: La altura debe ser un número positivo.\n');
else
    % Calcular el área
    area = (base * altura) / 2;

    % Mostrar el resultado
    fprintf('El área del triángulo con base %.2f y altura %.2f es: %.2f\n', base, altura, area);
end

